src/fprime_gds/common/data_types/ch_data.py

```diff
@@ -67,9 +67,7 @@ def _compute_display_text(self, val_obj, template):
         fmt_str = template.get_format_str()
         if temp_val is None:
             return ""
-        if fmt_str:
-            return format_string_template(fmt_str, (temp_val,))
-        return temp_val
+        return format_string_template(fmt_str, (temp_val,)) if fmt_str else temp_val
```

Comment:
I would argue that this is much less readable than before the change, and therefore not a better pattern. A succession of if statements clearly shows the priority order. Here we would have one if statement, then one if expression -- the reader kind of has to unfold that in his head and it takes a couple seconds to do so.

Reply:
As you want, there already is an if expr just above. But I respect your choice - you are the maintainer 😸

By the way, both files where you put a comment arenot black formatted, is it intended?

Reply:
Ah yes, I think there's a reason we're not running formatting checks on this repo, but we're going to want to fix that. Feel free to format those files you're modifying!
We want to avoid formatting the whole codebase at once to avoid large commits/PRs. So we're progressively formatting as things get modified.

Reply:
Done chef!

Reply:
Thank you very much 😄